<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenerateDialog</class>
 <widget class="QDialog" name="GenerateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>340</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>64</width>
    <height>48</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Generate a file</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="labelNbPeople">
       <property name="text">
        <string>Number of character(s)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="spinBoxNbPeople">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelNbText">
       <property name="text">
        <string>Number of sentence(s)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinBoxNbText">
       <property name="maximum">
        <number>99999</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelNbTrack">
       <property name="text">
        <string>Number of track(s)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="spinBoxNbTrack">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelNbLoop">
       <property name="text">
        <string>Number of loop(s)</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="spinBoxNbLoop">
       <property name="maximum">
        <number>99999</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelText">
       <property name="text">
        <string>Text </string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEditText"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="labelTimeCode">
       <property name="text">
        <string>TimeCode</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="PhTimeCodeEdit" name="lineEditTimeCode"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="labelInfoTxt">
       <property name="text">
        <string>Strip duration : </string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="labelInfoDuration">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="spaceBetweenTextLabel">
       <property name="text">
        <string>Space between text</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spaceBetweenTextSpinBox">
       <property name="maximum">
        <number>2400000</number>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>Run for tests</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PhTimeCodeEdit</class>
   <extends>QLineEdit</extends>
   <header>PhCommonUI/PhTimeCodeEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>buttonBox</tabstop>
  <tabstop>spinBoxNbPeople</tabstop>
  <tabstop>spinBoxNbText</tabstop>
  <tabstop>spaceBetweenTextSpinBox</tabstop>
  <tabstop>spinBoxNbTrack</tabstop>
  <tabstop>spinBoxNbLoop</tabstop>
  <tabstop>lineEditText</tabstop>
  <tabstop>lineEditTimeCode</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
